examples/hpc-slurm.yaml

```diff
@@ -21,6 +21,9 @@ vars:
   deployment_name: hpc-small
   region: us-central1
   zone: us-central1-a
+  # When set, active compute nodes will be cleaned up on destroy.
+  # Note that setting this option requires additional dependencies to be installed locally.
```

Comment:
I believe due to the `requires additional dependencies` this is intentionally left out of our examples.

I would propose the alternative action:
- do not change the blueprint
- In each slurm test (or at a minimum this slurm test) set `--vars enable_cleanup_compute=true`
- This will have the effect of turning it on in tests but leaving the example as is

You should be able to do this in the test definition: [example](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/6d3e6600e6e802f605a12336900da0294e40030e/tools/cloud-build/daily-tests/tests/hpc-enterprise-slurm.yml#L22-L24). Please test this out. Bools work on the actually command line but I don't know that his has been tested using this `cli_deployment_vars` in the test definition.

Reply:
+1

Reply:
After offline discussion, we decided to move the Ensure all nodes are powered down to always block in slurm integration test. 
Updated accordingly.

Reply:
I noted in offline discussion that there may be reasons to still go with my proposal here. I will remove my blocking request for change but would like to see the offline discussion resolve.

Reply:
Updated, after offline discussion we decided to keep both the solution. 

1) Add enable_cleanup_compute in the test file.
2) Move tasks from test partition to slurm integration test.